src/vm/util.cpp

```diff
@@ -2442,10 +2442,14 @@ size_t GetLargestOnDieCacheSize(BOOL bTrueSize)
     else
         return maxSize;
 
-#else
+#elif !defined(FEATURE_PAL)
 
     size_t cache_size = GetLogicalProcessorCacheSizeFromOS() ; // Returns the size of the highest level processor cache
     return cache_size;
+    
+#else
+    // TODO: This should be implemented in PAL
```

Comment:
Is it okay to return 0 here actually? I've been looking at how you would get this on Linux and it seems for platforms other than x86/AMD64 (already handled above), there is no standard way. 0 here should indicate unavailable, as per a comment in the header "Try to determine the largest last-level cache size of the machine - return 0 if unknown or no L2/L3 cache".